templates/server-statefulset.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "dctna.fullname" . }}-server
+  namespace: {{ .Release.Namespace }}
```

Comment:
For the stateful set you specify a namespace, for other resources not. Looks like inconsequent